flow/Makefile

```diff
@@ -678,16 +678,19 @@ skip_route: $(RESULTS_DIR)/4_cts.odb $(RESULTS_DIR)/4_cts.sdc
 	cp $(RESULTS_DIR)/3_place.odb $(RESULTS_DIR)/5_2_route.odb
 	cp $(RESULTS_DIR)/3_place.odb $(RESULTS_DIR)/6_1_fill.odb
 	touch $(RESULTS_DIR)/6_final.spef
-	$(MAKE) $(LOG_DIR)/6_report.log
-	$(MAKE) finish
 
 # To create a mock abstract quickly, good enough to iterate quickly on
 # floorplanning and detailed route at higher levels, run:
 #
 # make skip_cts
 # make skip_route
 # make generate_abstract
-generate_abstract: $(RESULTS_DIR)/6_final.gds $(RESULTS_DIR)/6_final.def  $(RESULTS_DIR)/6_final.v
+generate_abstract: $(LOG_DIR)/6_report.log \
+ $(RESULTS_DIR)/6_final.v \
+ $(RESULTS_DIR)/6_final.sdc \
+ $(GDS_FINAL_FILE) \
+ $(LOG_DIR)/6_report.log \
+ $(RESULTS_DIR)/6_final.def
```

Comment:
Why not use finish the dependency rather than this list?

Reply:
No reason, would it be better?

"finish" have some side-effects, printing some reports...

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/18087e3b5fbd26e712871437c22642720a7aafb7/flow/Makefile#L639

Reply:
It seems like an overly long list.  Even finish itself seems a bit redundant.  Generating GDS_FINAL_FILE seems like enough:
```
$(GDS_FINAL_FILE): $(GDS_MERGED_FILE)
$(GDS_MERGED_FILE): $(RESULTS_DIR)/6_final.def ...
$(RESULTS_DIR)/6_final.def: $(LOG_DIR)/6_report.log
```
etc

Reply:
I haven't whittled down the list to actually required targets. I just added more targets until generate_abstract worked.

generate_abstract is a bit funny: it doesn't actually have all the dependencies listed. I believe BLOCKS first runs a build, then runs generate_abstract. Perhaps BLOCKS should be changed not to run a build first, but to just call generate_abstract. We could then fix the errors and then this PR will be simpler.